.gitignore

```
@@ -109,5 +110,4 @@ dj_local_conf_old.json
 # emacs
 **/*~
 **/#*#
-**/.#*
-docker-compose.yml
+**/.#*
```

It is good practice to ignore the standard docker-compose files so that others can copy any environment they choose and modify as needed for local setups.

```suggestion
**/.#*
docker-compose.y*ml
```